docs/content/docs/try-flink-ml/quick-start.md

```diff
@@ -27,213 +27,64 @@ under the License.
 ```
 
-### Creating, Configuring, Training & Using Kmeans
-
-Flink ML classes for Kmeans algorithm include `KMeans` and `KMeansModel`.
-`KMeans` implements the training process of Kmeans algorithm based on the
-provided training data, and finally generates a `KMeansModel`.
-`KmeansModel.transform()` method encodes the Transformation logic of this
-algorithm and is used for predictions. 
-
-Both `KMeans` and `KMeansModel` provides getter/setter methods for Kmeans
-algorithm's configuration parameters. The example program explicitly sets the
-following parameters, and other configuration parameters will have their default
-values used.
-
-- `K`, the number of clusters to create
-- `seed`, the random seed to initialize cluster centers
-- `featuresCol`, name of the column containing input feature vectors
-- `predictionCol`, name of the column to output prediction results
+The command above would submit and execute Flink ML's `KMeansExample` job. There
+are also example jobs for other Flink ML algorithms, and you can find them in
+`flink-ml-examples` module.
 
-When the program invokes `KMeans.fit()` to generate a `KMeansModel`, the
-`KMeansModel` will inherit the `KMeans` object's configuration parameters. Thus
-it is supported to set `KMeansModel`'s parameters directly in `KMeans` object.
-
-```java
-KMeans kmeans = new KMeans()
-        .setK(2)
-        .setSeed(1L)
-        .setFeaturesCol(featuresCol)
-        .setPredictionCol(predictionCol);
-
-KMeansModel model = kmeans.fit(input);
-
-Table output = model.transform(input)[0];
+A sample output in your terminal is as follows.
 
 ```
+Features: [9.0, 0.0]    Cluster ID: 1
+Features: [0.3, 0.0]    Cluster ID: 0
+Features: [0.0, 0.3]    Cluster ID: 0
+Features: [9.6, 0.0]    Cluster ID: 1
+Features: [0.0, 0.0]    Cluster ID: 0
+Features: [9.0, 0.6]    Cluster ID: 1
 
-### Collecting Prediction Result
-
-Like all other Flink programs, the codes described in the sections above only
-configures the computation graph of a Flink job, and the program only evaluates
-the computation logic and collects outputs after the `execute()` method is
-invoked. Collected outputs from the output table would be `Row`s in which
-`featuresCol` contains input feature vectors, and `predictionCol` contains
-output prediction results, i.e., cluster IDs.
-
-```java
-for (CloseableIterator<Row> it = output.execute().collect(); it.hasNext(); ) {
-    Row row = it.next();
-    DenseVector vector = (DenseVector) row.getField(featuresCol);
-    int clusterId = (Integer) row.getField(predictionCol);
-    System.out.println("Vector: " + vector + "\tCluster ID: " + clusterId);
-}
 ```
 
-```
-Vector: [0.3, 0.0]	Cluster ID: 1
-Vector: [9.6, 0.0]	Cluster ID: 0
-Vector: [9.0, 0.6]	Cluster ID: 0
-Vector: [0.0, 0.0]	Cluster ID: 1
-Vector: [0.0, 0.3]	Cluster ID: 1
-Vector: [9.0, 0.0]	Cluster ID: 0
-```
+You have successfully ran a Flink ML job. Now please feel free to explore other
```

Comment:
Is this paragraph copied and pasted from other places? It seems that there is no further section in this document.

Reply:
I agree that it might be ambiguous. I'll remove this line.